src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -202,7 +202,7 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 				.addControl('name', '', Validators.required)
 				.addControl('description', '', Validators.required)
 				.addControl('escoID', '')
-				.addControl('studyLoad', 60, [Validators.required, this.positiveInteger, Validators.max(1000)])
+				.addControl('studyLoad', 60, [Validators.required, this.positiveInteger, Validators.max(1000000)])
```

Comment:
Why is the limit of 1000000 only here and not in line 196 as well?

Reply:
Added a comment to explain

Reply:
The comment explains why the limit is *here*, but not why it's *not* in line 196, so for the AI competencies. Shouldn't the limit apply both to "normal", as well as for AI competencies?

Reply:
The only reason i added this limit to normal competencies was that after adding a normal competency the number is displayed in this white box: 
![limit](https://github.com/mint-o-badges/badgr-ui/assets/65068389/76d966dd-2bd4-4af0-a045-af8f3d4cece3)
If you add a number with 500 characters this could break the ui here, this issue does not exist for ai competencies

Reply:
Hm ok, I still think it would be nicer if it was consistent. But I guess it's unlikely anyone's gonna try to enter such a high value anyway, so I'm also ok if you don't want to add the limit for AI competencies as well.